tests/scripts/lua/management/lib/swagger.lua

```diff
@@ -31,6 +31,9 @@ describe('Testing v2 management interface', function()
         "name": "Hello World API",
         "resources": {
           "/hello": {
+            "cors": {
+              "origin": "true"
```

Comment:
@mdeuser if you're using the v2 interface, you should set `cors.enabled` in the `x-gateway-configuration` block:
```
"x-gateway-configuration": {
  "cors": {
     "enabled": true
   },
   ...
}
```
What you're seeing is how the Swagger doc is converted behind the scenes to what the gateway understands (and also how it is handled in the v1 interface).

I have a PR open for the v2 interface docs which should clear things up more: https://github.com/openwhisk/openwhisk-apigateway/pull/190